cypress/e2e/orders/receive-piece-from-order.cy.js

```diff
@@ -48,7 +48,7 @@ describe('orders: Receive piece from Order', () => {
       .then(orderNumber => {
         cy.visit(TopMenu.ordersPath);
         Orders.searchByParameter('PO number', orderNumber);
-        Helper.selectFromResultsList();
+        Orders.selectFromResultsList(orderNumber);
         Orders.openOrder();
         InteractorsTools.checkCalloutMessage(`The Purchase order - ${orderNumber} has been successfully opened`);
         Orders.receiveOrderViaActions();
```

Comment:
this tests fails: did not find accordion "Holdings: Main Library >", did you mean one of:

Reply:
Generally Great job!!!